src/item.cpp

```diff
@@ -8197,40 +7979,15 @@ void item::fill_with( item &liquid, int amount )
         return;
     }
 
-    if( !is_container() ) {
-        if( !is_reloadable_with( liquid.typeId() ) ) {
-            debugmsg( "Tried to fill %s which is not a container and can't be reloaded with %s.",
-                      tname(), liquid.tname() );
-            return;
-        }
-        ammo_set( liquid.typeId(), ammo_remaining() + amount );
-    } else if( is_food_container() ) {
-        // if container already has liquid we need to sum the energy
-        item &cts = contents.front();
-        const float lhs_energy = cts.get_item_thermal_energy();
-        const float rhs_energy = liquid.get_item_thermal_energy();
-        if( rhs_energy < 0 ) {
-            debugmsg( "Poured item has no defined temperature" );
-        }
-        const float combined_specific_energy = ( lhs_energy + rhs_energy ) / ( to_gram(
```

Comment:
this looks suspicious. idk what was going through my head when i deleted this

Reply:
looking at it again, this handles stacking of items which i do not currently handle anyway, so it should be fine. there needs to be some kind of item::combine() that combines one with another if possible.